sites.json

```diff
@@ -1,5 +1,15 @@
 [
     {
+        "name": "1Password",
+        "url": "http://my.1password.com/profile",
```

Comment:
The URL is correct but the scheme should be https. Forgot it when amending sites.json. Beg my pardon!